winui/AvatarView/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of WinUI AvatarView control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# WinUI AvatarView Overview
+
+The .NET AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
```

Comment:
The .NET AvatarView control provides a graphical representation of a user image that allows you to customize the view by adding an image, background color, icon, text, and more.